Setup/Product.wxs

```diff
@@ -599,14 +589,6 @@
               - The add-in feature state (!feature-state) for the Visual Studio version is installed (3 = INSTALLSTATE_LOCAL). See:
                 Conditional Statement Syntax (http://msdn.microsoft.com/en-us/library/aa368012.aspx)
             -->
-            <Custom Action="RemoveVS2005AddInCommands"
-                 After="RemoveFiles">Installed AND NOT UPGRADINGPRODUCTCODE AND VS2005INSTFOLDER
-                                     AND !VS2005FeatureId=3</Custom>
```

Comment:
I am not sure. If there are two VS installed 2008 and 2010. Do we need to uninstall addin for 2008 to be able to install 2010 one?